config/serverless.chat.yml

```diff
@@ -0,0 +1,41 @@
+# Chat Project Config
+
+## Enable the Serverless Chat plugin
+xpack.serverless.chat.enabled: true
+
+## Cloud settings
+xpack.cloud.serverless.project_type: search
+
+## Set the home route
+uiSettings.overrides.defaultRoute: /app/home
+
+## Disable plugins that belong to other solutions
```

Comment:
having to disable these plugins on a per-plugin basis is rather brittle. is my understanding correct that we don't have the ability to disable all solution plugins currently? if so, i can create a github issue to track this if it doesn't exist.

Reply:
We have an issue for that https://github.com/elastic/kibana/issues/204227
In the meantime, I thought it'd be interesting to start with as many plugins disabled as possible, and gradually adding what we miss.

Reply:
Q: should we have this list in the base `serverless.yml` and let each solution enable what they need instead?